sf-docker/multi-camera/docker-compose.yml

```diff
@@ -274,6 +294,44 @@ services:
     # - "/var/tmp/innovatrics/tensor-rt:/var/tmp/innovatrics/tensor-rt"
     #runtime: nvidia
 
+  object-detector:
+    image: ${REGISTRY}sf-object-detector:${SF_VERSION}
+    container_name: SFObjectDetectorCpu
+    restart: unless-stopped
+    environment:
+      - RabbitMQ__Hostname
+      - RabbitMQ__Username
+      - RabbitMQ__Password
+      - RabbitMQ__Port
+      - AppSettings__Log_RollingFile_Enabled=false
+      - AppSettings__USE_JAEGER_APP_SETTINGS
+      - JAEGER_AGENT_HOST
+      # - Gpu__GpuEnabled=true
+      # - Gpu__GpuNeuralRuntime=Tensor
```

Comment:
one more think came to mind:
TensorRT is not available for `object-detector` and `pedestrian-extractor` so maybe we should remove these commented-out lines